electron/process_manager.ts

```diff
@@ -363,7 +337,7 @@ function startTun2socks(host: string, onDisconnected: () => void): Promise<void>
 
       resolve();
     } catch (e) {
-      reject(e);
+      reject(new errors.ShadowsocksStartFailure(`could not start tun2socks`));
```

Comment:
I think `errors.VpnStartFailure` may be more appropriate here.

Reply:
I chose this error because the main reason I can imagine tun2socks failing for is antivirus software and this message specifically points the user in that direction. I added a comment to explain this choice.